test/verifier.integration.spec.js

```diff
@@ -156,13 +156,7 @@ describe("Verifier Integration Spec", function () {
 							pactBrokerUsername: 'foo',
 							pactBrokerPassword: 'baaoeur'
 						});
-						return verifier.verify()
-							.then(function () {
-								throw new Error("Expected an error but got none");
-							})
-							.catch(function (err) {
-								expect(err.message).to.contain("Unauthorized")
-							})
+						return expect(verifier.verify()).to.eventually.be.rejected
```

Comment:
This was required as the behaviour slightly changed in the new verifier although functionally remained the same. This was much simpler and less fragile.